plugin/evm/block_builder.go

```diff
@@ -169,11 +169,17 @@ func (b *blockBuilder) awaitSubmittedTxs() {
 					// Give time for this node to build a block before attempting to
 					// gossip
 					time.Sleep(waitBlockTime)
-					// [GossipTxs] will block unless [gossiper.txsToGossipChan] (an
-					// unbuffered channel) is listened on
-					if err := b.gossiper.GossipTxs(ethTxsEvent.Txs); err != nil {
+
+					// Retrieve nodes to gossip to
```

Comment:
For the first iteration, we should add a config parameter in `plugin/evm/config.go` that determines whether subnet-evm uses the existing gossip algorithm or targeted gossip.